src/app/functions.tsx

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../types/Todo';
+import { TodoStatus } from '../types/Status';
+
+export const filterTodos = (
+  todos: Todo[],
+  status: TodoStatus,
+  query: string,
+): Todo[] => {
+  let filteredTodos;
+
+  switch (status) {
+    case 'active':
+      filteredTodos = todos.filter(todo => !todo.completed);
+      break;
+    case 'completed':
```

Comment:
Not fixed 
<img width="905" alt="image" src="https://github.com/user-attachments/assets/06bc7dd2-d2c4-4fba-bd94-8dbefaf76e42">